File: Python/Program/Business.py
class Business:
    def __init__(self, owner: str, name: str, address: str):
        self._owner = owner
        self._name = name
        self._address = address

    def get_owner(self) -> str:
        return self._owner

    def set_owner(self, owner: str):
        self._owner = owner

    def get_name(self) -> str:
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_address(self) -> str:
        return self._address

    def set_address(self, address: str):
        self._address = address

File: C++/Program/Business.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Business
{
private:
    string owner;
    string name;
    string address;

public:
    // Constructor
    Business() {}

    Business(string owner, string name, string address)
    {
        this->owner = owner;
        this->name = name;
        this->address = address;
    }

    // Getter and Setter for owner
    string getOwner()
    {
        return owner;
    }

    void setOwner(string owner)
    {
        this->owner = owner;
    }

    // Getter and Setter for name
    string getName()
    {
        return name;
    }

    void setName(string name)
    {
        this->name = name;
    }

    // Getter and Setter for address
    string getAddress()
    {
        return address;
    }

    void setAddress(string address)
    {
        this->address = address;
    }
};